api/alert_channels_aws_cloudwatch.go

```diff
@@ -0,0 +1,45 @@
+//
+// Author:: Vatasha White (<vatasha.white@lacework.net>)
+// Copyright:: Copyright 2021, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package api
+
+// GetAWSCloudwatch gets a single instance of an AWS Cloudwatch alert channel
+// with the corresponding integration guid
+func (svc *AlertChannelsService) GetAWSCloudwatch(guid string) (response AWSCloudwatchAlertChannelResponseV2, err error) {
```

Comment:
Likewise for the function names, we try to follow the pattern `Get<Type>()`
```suggestion
func (svc *AlertChannelsService) GetCloudwatchEb(guid string) (response AWSCloudwatchAlertChannelResponseV2, err error) {
```